### PR TITLE
[Copy] Change to sentence case; remove superfluous colons; update English and French copy

### LINKED ISSUE #12335: 🔤 Small copy fixes talent request details admin side
## 🔤 Copy

Some copy fixes on the talent request details pages shown to the communities/admins. Primarily issues of title case to sentence case, but also some other fixes and some French edits.

Please also remove all the colons from the labels in both languages.

## 🕵️ Details

On the page with the request details (/admin/talent-requests/IDnumber), please update:
-	“Manager Information” to “Manager information”
-	“Government Email” to “Government email”
-	“Date Received” to “Date received”
-	“Request Information” to “Request information”
-	“Conditions of employment / Operational requirements” to “Conditions of employment or operational requirements”
-	“Candidate Results” to “Candidate results”
- "Candidate Name" to "Candidate name"
-	“These notes are available to all managers of this pool, but not to candidates.” to “These notes are only available here and will not be shared with the candidates.”

image
image
image

**French**:
-	“Courriel du conseiller/ère en RH » to « Courriel du (de la) conseiller(-ère) en RH»
-	« Conditions d’emploi/Exigences opérationnelles » to « Conditions d’emploi ou exigences opérationnelles »
-	« Résultats du candidat » to « Résultats des candidates et candidats »
- « Nom du candidat » to « Nom du (de la) candidat(e) »
-	« Notes sur les demandes » to « Notes sur cette demande »
-	« Ces notes sont accessibles à tous les gestionnaires de ce bassin, mais pas aux candidats. » to  « Ces notes sont uniquement disponibles ici et ne seront pas communiquées aux candidates et candidats. »
-	(Button) « Enregistrer les notes » to « Enregistrez les notes » 
-	(Button) « Sauvegarder le changement d’état » to « Sauvegardez le changement d'état »

image
image
image
image

Please also remove all the colons from the labels in both languages.

## ✅ Acceptance Criteria

- [ ] Copy edits have been implemented as listed under details
- [ ] Colons have been removed from the labels

### PR DESCRIPTION
**Pull request by mnigh**
🤖 Resolves #12335.

## 👋 Introduction

This PR makes some copy fixes on the search request page and view request page, primarily issues of title case to sentence case, but also some other fixes to English copy and some French edits. It also removes all the colons from the labels in both languages (`FilterBlock` component) and updates some other text for consistency.

## 🧪 Testing

1. `pnpm build:fresh`
2. Navigate to Search request page
3. Verify English and French copy matches requested changes in linked issue
4. Navigate to View request page
5. Verify English and French copy matches requested changes in linked issue

## 📸 Screenshots

<details><summary>Search request page</summary>

### English
localhost_8000_en_search_request

### French
localhost_8000_fr_search_request
</details> 

<details><summary>View request page</summary>

### English
localhost_8000_en_admin_talent-requests_f32bf6c5-a890-4e8e-8326-57d9e38050c3

### French
localhost_8000_fr_admin_talent-requests_f32bf6c5-a890-4e8e-8326-57d9e38050c3
</details> 

<details><summary>Other</summary>
Screen Shot 2024-12-24 at 13 53 49
Screen Shot 2024-12-24 at 13 55 00
Screen Shot 2024-12-24 at 13 55 31
</details>

### CODE REVIEW

**Review by esizer — APPROVED** [review]
LGTM! 👍

**Review by NienkeBr — COMMENTED** [review]
I noticed one more point I'd like to have updated, if I can still add it here.
On the search request page, could we change "Department/Hiring organization" and "Ministère/organisation d'embauche" to "Department or hiring organization" and "Ministère ou organisation d'embauche"?
image
image

**Comment by mnigh** [comment]
> I noticed one more point I'd like to have updated, if I can still add it here. On the search request page, could we change "Department/Hiring organization" and "Ministère/organisation d'embauche" to "Department or hiring organization" and "Ministère ou organisation d'embauche"?

## Screenshots

localhost_8000_en_search

localhost_8000_fr_search_request

File: apps/web/src/lang/fr.json [hunk 1]
```diff
@@ -203,6 +203,10 @@
     "defaultMessage": "Sécurité de la TI",
     "description": "Label for the 'security' IT work stream"
   },
+  "/3mqz9": {
+    "defaultMessage": "Renseignements sur le gestionnaire",
```
[the remark this answers is not in the document]
Comment (mnigh) [reply apps/web/src/lang/fr.json:207]: Oops, nice catch.

## Screenshot
localhost_8000_fr_admin_talent-requests_1856acaf-4d86-4668-86d5-1b6952bc867f

**Review by NienkeBr — APPROVED** [review]

**Review by esizer — APPROVED** [review]

**Pull request merged**